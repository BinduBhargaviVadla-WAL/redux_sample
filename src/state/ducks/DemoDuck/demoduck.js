const DemoDuck = () => {
  return <div>This is demo-duck</div>;
};
export default DemoDuck;
