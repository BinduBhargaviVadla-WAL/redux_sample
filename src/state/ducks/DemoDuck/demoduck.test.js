import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import DemoDuck from "./demoduck";

describe("demo testing", () => {
  test("testing snapshot for DemoDuck Component", () => {
    const component = renderer.create(<DemoDuck />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test("testing DemoDuck Component", () => {
    render(<DemoDuck />);
    const textElement = screen.getByText(/This is demo-duck/i);
    expect(textElement).toBeInTheDocument();
  });
});
