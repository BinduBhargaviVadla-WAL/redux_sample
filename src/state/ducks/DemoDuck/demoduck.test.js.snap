// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demo testing testing snapshot for DemoDuck Component 1`] = `
<div>
  This is demo-duck
</div>
`;
